superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx

```diff
@@ -69,13 +70,14 @@ interface ChartHolderProps {
   setFullSizeChartId: (chartId: number | null) => void;
   isInView: boolean;
 }
-
+const theme=useTheme()
```

Comment:
Looks like this (and the  style const below it) might need to go within the function component... or there are other workarounds for that. Holler if you want any help here, I _might_ be able to push commits to your fork/branch